lib/mix2nix.ex

```diff
@@ -115,7 +117,7 @@ defmodule Mix2nix do
 		      version = "#{version}";
 
 		      src = fetchHex {
-		        pkg = "${name}";
+		        pkg = "#{hex_name}";
```

Comment:
Define `name` on line 116 to be the `hex_name`. Then, keep the src block the same.